addons/anonymization/wizard/anonymize_wizard.py

```diff
@@ -264,8 +261,8 @@ def reverse_anonymize_database(self):
             raise UserError('%s: %s' % (_('Error !'), _("It is not possible to reverse the anonymization process without supplying the anonymization export file.")))
 
         # reverse the anonymization:
-        # load the pickle file content into a data structure:
-        data = pickle.loads(base64.decodestring(self.file_import))
+        # load the json file content into a data structure:
+        data = json.loads(self.file_import)
```

Comment:
Backward-compatibility: check the first bytes to see whether it is a pickle (generated before 11.0), and decode with the appropriate tool.